com.playeveryware.eos/Runtime/Core/Config/Config.cs

```diff
@@ -239,8 +285,11 @@ public static async Task<T> GetAsync<T>() where T : Config
             s_cachedConfigs.Add(typeof(T), instance);
 #endif
 
-            // Call prepare function.
-            instance.MigrateConfig();
+            if (instance.NeedsMigration())
+            {
+                instance.MigrateConfig();
+                await instance.WriteAsync();
```

Comment:
`WriteAsync` and `Write` are both defined to only be included in the editor. Trying to compile a standalone build, or selecting "Player" as the project, will show that this isn't included, and fail to build.

Reply:
Nice catch, I've fixed this now with compile-time conditionals.